### ISSUE TITLE
[BUG] Possible regression: pinching not working with wheelzoomtool

### ISSUE THREAD

**Issue by solstag** [bokeh/bokeh#12731]
### Software versions

Python version      :  3.9.9 (main, Jan  1 1970, 00:00:01) 
IPython version     :  8.2.0
Tornado version     :  5.1.1
Bokeh version       :  3.0.3
BokehJS static path :  /home/eris/.local/lib/python3.9/site-packages/bokeh/server/static
node.js version     :  (not installed)
npm version         :  (not installed)
Operating system    :  Linux-6.0.16-x86_64-with-glibc2.33

### Browser name and version

Different behaviors in Firefox (108.0.1 (64-bit)) and Webkit (WebKitGTK 2.36.8)

### Expected behavior

Active Wheelzoom should enable zoom on pinch and avoid the browsers default action of zooming the viewport.

In other words: zooming YES, viewport zoomed NO.

### Observed behavior

On Firefox (108.0.1 (64-bit)): zooming NO, viewport zoomed YES.

On Webkit (WebKitGTK 2.36.8): zooming YES, viewport zoomed YES.

### Example code

```Python
from bokeh.models import ColumnDataSource
from bokeh.plotting import figure, output_file, save

cds = ColumnDataSource(
    dict(
        x=[0],
        y=[0],
        w=[1],
        h=[1],
    )
)
fig = figure(
    x_range=(-2, 2),
    y_range=(-2, 2),
    width=800,
    height=600,
    tools=['ywheel_zoom'],
    active_scroll="ywheel_zoom",
)
fig.rect(
    x="x",
    y="y",
    width="w",
    height="h",
    source=cds,
)
output_file("test_pinch_screen.html", mode="inline")
save(fig)
```

**Comment by solstag** [comment]
Filed upon request at https://discourse.bokeh.org/t/enabling-pinch-zoom-on-a-touchscreen/9952/3

**Comment by mattpap** [comment]
I noticed the problem when working on resizable box annotations. This should be fixed in PR #12468 (https://github.com/bokeh/bokeh/pull/12468/commits/122591feea375b93ed45424f563e901eeb693260), though I need to confirm this.
